FE/baseball/src/Components/Game/ScoreBoard/ScoreBoard.jsx

```diff
@@ -0,0 +1,39 @@
+import ScoreTable from './ScoreTable/ScoreTable';
+import { ScoreBoardStyles as S } from './ScoreBoardStyles';
+import { useState } from 'react';
+import PopUpButton from './PopUpButton';
+
+const ScoreBoard = () => {
+  const initialScoreBoardPosition = -199;
```

Comment:
상수는 snake case로 명명해주기도 합니다 ㅎㅎ
```suggestion
  const INITAL_SCORE_BOARD_POSITION = -199;
```